plutus-core/plutus-core/src/PlutusCore/Evaluation/Machine/CostModelInterface.hs

```diff
@@ -114,41 +106,42 @@ applyParams cm params = case toJSON cm of
 -- | Parameters for a machine step model and a builtin evaluation model bundled together.
 data SplitCostModelParams =
     SplitCostModelParams {
-      machineParams :: CostModelParams
-    , builtinParams :: CostModelParams
+      _machineParams :: CostModelParams
```

Comment:
I thought I'd better name these fields to avoid confusion, but the names aren't actually used anywhere else.